app/src/main/res/values/strings.xml

```diff
@@ -78,7 +78,7 @@
 
     <string name="superuser_toggle_notification">Notifications</string>
     <string name="superuser_toggle_revoke">Revoke</string>
-    <string name="superuser_policy_none">No app has asked for superuser permission yet.</string>
+    <string name="superuser_policy_none">No apps have asked for superuser permission.</string>
```

Comment:
```suggestion
    <string name="superuser_policy_none">No apps have asked for superuser permission yet.</string>
```

Reply:
Would need to be "permissions", no?

Reply:
I think you can only get away with that in English.
In other languages it necessitates there being more than one permission for each app.

Reply:
Yeah fair enough, English is quite loose in a lot of ways. 😛